packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/WholeMeetingSummary.tsx

```diff
@@ -49,14 +41,12 @@ const WholeMeetingSummary = (props: Props) => {
     meetingRef
   )
   if (meeting.__typename === 'RetrospectiveMeeting') {
-    const {summary: wholeMeetingSummary, reflectionGroups, phases} = meeting
-    const discussPhase = phases!.find((phase) => phase.phaseType === 'discuss')
-    const {stages} = discussPhase ?? {}
-    const hasTopicSummary = reflectionGroups!.some((group) => group.summary)
-    const hasDiscussionSummary = !!stages?.some((stage) => stage.discussion?.summary)
-    const hasOpenAISummary = hasTopicSummary || hasDiscussionSummary
-    if (!hasOpenAISummary) return null
-    if (hasOpenAISummary && !wholeMeetingSummary) return <WholeMeetingSummaryLoading />
+    const {summary: wholeMeetingSummary, reflectionGroups, organization} = meeting
+    const hasMoreThanOneReflection =
+      (reflectionGroups?.length && reflectionGroups.length > 1) ||
+      reflectionGroups?.some((group) => group.reflections.length > 1)
+    if (!hasMoreThanOneReflection || organization.featureFlags.noAISummary) return null
```

Comment:
If there's more than one reflection, we'll always create the meeting summary unless `noAISummary` is true, so we don't need to check if there's a discussion summary or reflection group summary (which is now removed anyway)